package.json

```diff
@@ -31,10 +31,12 @@
   },
   "dependencies": {
     "body": "^5.0.0",
+    "commander": "^2.6.0",
```

Comment:
No that's not possible. The only other way I could think of is to create a separate npm-module (e.g. testpop) in a subdirectory that requires ringpop using a relative path. 

Also, main.js is currently installed as the `ringpop`-binary (see https://github.com/uber/ringpop-node/blob/master/package.json#L17) which means that ringpop currently not only depends on commander during testing. So we should either add commander as a dependency or stop providing the ringpop-binary.